Code/Framework/AzCore/AzCore/Math/SimdMath.h

```diff
@@ -21,6 +21,11 @@
 #   endif
 #endif
 
+#undef AZ_TRAIT_USE_PLATFORM_SIMD_SSE
+#undef AZ_TRAIT_USE_PLATFORM_SIMD_NEON
+#undef AZ_TRAIT_USE_PLATFORM_SIMD_SCALAR
+#define AZ_TRAIT_USE_PLATFORM_SIMD_SCALAR 1
+
```

Comment:
Is this debug code or intentional?

Reply:
Debug code to force AZ::Simd into scalar mode without messing around in the PAL traits. It's already removed.